README.md

```diff
@@ -41,30 +42,27 @@ _Theodore Levitt_ once said:
 ### Goals of Submarine
 
 #### Model Training (Experiment)
-
-- Can run experiment (training jobs) on prem, on cloud. Via easy-to-use User-Interfaces
-- Easy for Data-Scientist (DS) to manage training code and dependencies (Docker, Python Dependencies, etc.) .
-- ML-focused APIs to run/track experiment from Python SDK (notebook), REST API, and CLI.
-- Provide APIs to run training jobs by using popular frameworks (Standalone/Distributed TensorFlow/PyTorch/Hovorod).
-- Pre-packaged Training Template for Data-Scientists to focus on domain-specific tasks (like using DeepFM to build a CTR prediction model).
-- Support GPU and other compute speed-up devices.
-- Support running on K8s/YARN or other resource management system.
+- Run/Track distributed training `experiment` on prem or cloud via easy-to-use UI/ API/ SDK.
```

Comment:
```suggestion
- Run/Track distributed training `experiment` on prem or cloud via easy-to-use UI/API/SDK.
```

Reply:
@jiwq Thank you for this recommendation! I have fixed it.